src/App.tsx

```diff
@@ -1,26 +1,234 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import {
+  USER_ID,
+  addTodos,
+  deleteTodos,
+  getTodos,
+  updateTodos,
+} from './api/todos';
+import { Header } from './Components/Header/Header';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Errors } from './Components/Errors/Errors';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(Status.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [inputDisabled, setInputDisabled] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState<number[]>([]);
```

Comment:
```suggestion
  const [loadingTodoIds, setIsLoading] = useState<number[]>([]);
```
or `processingTodoIds`

isLoading sounds like boolean value which is not true in this case